File: 8_kyu/Transportation_on_vacation.r
# Returns the cost to rent a car for d days.
rental_car_cost <- function(d){
  # $40 per day
  # more than 3d: $20 discount
  # more than 7d: total $50 discount
  40*d - 20*(d >= 3) - 30*(d >= 7)
}

File: 8_kyu/Transportation_on_vacation.py
def rental_car_cost(d: int) -> int:
    # base: $40 per day
    cost = 40 * d;
    if d >= 7:
        # more then 7 day: $50 discount
        cost -= 50
    elif d >= 3:
        # more then 3 day: $20 discount
        cost -= 20
    return cost